src/styles/sidebar/components/group-list-section.scss

```diff
@@ -0,0 +1,18 @@
+/* The groups section. */
+
+.dropdown-menu__row {
```

Comment:
This should go in the file for the `dropdown-menu` component, assuming we have one.

Reply:
I added a dropdown-menu__row--no-border in common with the other dropdown-menu__row css classes.